treeherder/config/settings.py

```diff
@@ -21,6 +22,8 @@
 # Top Level configuration
 DEBUG = env.bool("TREEHERDER_DEBUG", default=False)
 
+IS_WINDOWS = "windows" in platform.system().lower()
```

Comment:
I would set this directly in `test/conftest.py` since it only is related to running tests. Up to you.